assets/lang/it_IT.ini

```diff
@@ -653,7 +653,7 @@ Texture Replacement = Sostituzione delle texture
 Toggle Fullscreen = Passa a Schermo Intero
 Toggle mode = Modalità Scambio
 Triangle = Triangolo
-Unthrottle = Non velocizzare
+Fast-forward = Non velocizzare
```

Comment:
Should be `Fast-forward = Velocizza`, as this is literally "Don't speed up"

Reply:
Yeah, it seems Unthrottle is just very confusing and several languages had opposite translations.

I've fixed them using Google Translate, hopefully they'll at least be in the right direction...